zebra/zebra_nhg.c

```diff
@@ -256,7 +256,7 @@ struct nhg_hash_entry *zebra_nhg_find(struct nexthop_group *nhg,
 			 */
 			flog_warn(
 				EC_ZEBRA_DUPLICATE_NHG_MESSAGE,
-				"Nexthop Group from kernel with ID (%d) is a duplicate, ignoring",
+				"Nexthop Group from with ID (%d) is a duplicate, ignoring",
```

Comment:
Probably want to remove the "from" as well in this change.

Reply:
This has already been addressed:
```
        flog_warn(EC_ZEBRA_DUPLICATE_NHG_MESSAGE,                                
                  "Nexthop Group with ID (%d) is a duplicate, ignoring",         
                  nhe->id);    
```
in `zebra_nhg_set_dup()`

I believe I addressed all of @mjstapp 's comments as well.